portfolio/src/main/webapp/index.html

```diff
@@ -4,20 +4,56 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat|Poppins">
+    <script src="script.js"></script>    
   </head>
   <body>
-    <div id="content">
-      <h1>Himani's Portfolio</h1>
-      <p>Hi, I'm Himani! Welcome to my portfolio.</p>
-      <img src="/images/me.jpeg" alt="Himani's face" width="250" height="250">
-      <p>I'm a rising junior at the University of Georgia. I study Computer Science and Cognitive Science.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <p>Click here for a random fact about me:</p>
-      <button onclick="addRandomFact()">❓</button>
-      <div id="fact-container"></div>
+    <div class="content">
+      <div class="header">
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="#about">About</a>
+          <a href="gallery.html">Gallery</a>
+          <a href="step.html">STEP</a>
+          <a href="#footer">Contact</a>
+        </nav>
+      </div>
+
+      <div class="intro">
+        <h3 onclick="addRandomGreeting()" id="greeting-container">HELLO,  I'M</h3> 
+        <h1>Himani Yadav.</h1>
+        <h2>Computer Science &amp; Cognitive Science Student</h2>
+        <img src="/images/me.jpeg" alt="Himani's face" width="250" height="250">
+      </div>
+
+      <div class="about">
+        <h2 id="about">About Me</h2>
+        <p>
+           Hi! I'm a rising junior at the University of Georgia. I first found an interest in code
+           in middle school, as I attempted to create custom CSS themes for my Tumblr page. 
+           But, in high school, as I served as captain of my Mock Trial team, I convinced that I wanted to be a lawyer. 
+           I rediscovered my passion for tech late in my junior year of high school, when I joined the Girls Who Code club.
+           With my double majors, I hope to apply knowledge from disciplines such as computer science, 
+           linguistics, philosophy, and psychology to focus on the burgeoning field of Artificial Intelligence.<br/><br/>
+           When I'm not in front of a computer, you can probably find me exploring the eats around Athens, 
+           singing, crafting, or curled up with a good book.
+        </p>
+        <p>Click here for a random fact about me:</p>
+        <button onclick="addRandomFact()">Fun Fact!</button>
+        <div id="fact-container"></div>
+      </div>
+
+      <div class="footer">
+        <h2>Get in touch.<br/>
+          <p><a href="mailto:himaniyadav@gmail.com" target="_blank" style="font-family:Montserrat; font-size:15px; font-weight:400">himaniyadav@gmail.com</a> <br/>
```

Comment:
I know that the instructions say that "Learning CSS is optional for STEP", however I wanted to ask if wanted to actually learn and use CSS. I'm asking because applying inline style attributes is generally to be avoided whenever possible; it's preferable to use classes and CSS to apply the style. This is for several reasons:
a) They are more specific than anything else i.e. if this line also has a class with a specified font-size, it would be overridden by the inline style (Here's an overview: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)
b) More difficult to view the styles at a glance as you have to dig through the html to figure it out
c) Harder to refactor since it'd be scattered throughout the codebase

Reply:
Okay, thanks! I am doing CSS as well, and I think I specified similar attributes in the "footer" class in the CSS file. I didn't catch that I left the inline style attribute in the HTML. I'll fix that!

Reply:
Oops, I didn't yet see that you had added and are using the style.css until the last commit. I take that to mean you do know about it and are willing to use it, therefore I recommend adding a class on this anchor tag and applying the styles in css